### PR TITLE
feat: filesyncer sidecar for workflows

### PR DESCRIPTION
**Pull request by Vafilor**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes onepanelio/core#

**Special notes for your reviewer**:

**Checklist**

Please check if applies

- [ ] I have added/updated relevant unit tests
- [ ] I have added/updated relevant documentation

Required 

- [x] I accept to release these changes under the Apache 2.0 License

### CODE REVIEW

File: pkg/workflow_execution.go [hunk 1]
```diff
@@ -2063,6 +2067,38 @@ func getCURLNodeTemplate(name, curlMethod, curlPath, curlBody string, inputs wfv
 	return
 }
 
+func injectFilesyncerSidecar(wf *wfv1.Workflow) error {
+	filesyncer := wfv1.UserContainer{
+		Container: corev1.Container{
+			Name:  "sys-filesyncer",
+			Image: "onepanel/filesyncer:test-server-6",
```
Comment (rushtehrani) [inline pkg/workflow_execution.go:2074]: Commenting here so we don't forget to change this before merge

**Review by rushtehrani — CHANGES_REQUESTED** [review]

File: pkg/workflow_execution.go (hunk 1 above)
Comment (rushtehrani) [reply pkg/workflow_execution.go:2074]: @Vafilor want to update this to `onepanel/filesyncer:v0.19.0`?

File: pkg/workflow_execution.go [hunk 2]
```diff
@@ -2063,6 +2067,38 @@ func getCURLNodeTemplate(name, curlMethod, curlPath, curlBody string, inputs wfv
 	return
 }
 
+func injectFilesyncerSidecar(wf *wfv1.Workflow) error {
+	filesyncer := wfv1.UserContainer{
+		Container: corev1.Container{
+			Name:  "sys-filesyncer",
+			Image: "onepanel/filesyncer:test-server-7",
```
Comment (rushtehrani) [inline pkg/workflow_execution.go:2074]: I think we can update this to `v0.19.0` @Vafilor I'm creating an image for it.

Comment (Vafilor) [reply pkg/workflow_execution.go:2074]: @rushtehrani Roger that.

**Pull request merged**